cmdutil/closureLogic.go

```diff
@@ -14,8 +14,8 @@ const (
 	completedButConnErrorsExitStatus = 2
 )
 
-func CloseNicely(host string, port int, gc mtcpclient.GroupOfConnections) {
-	printClosureReport(host, port, gc)
+func CloseNicely(host string, port int, gc *mtcpclient.GroupOfConnections) {
```

Comment:
why are we passing this by reference rather than value? We are not modifying any content, right? If you are concerned about performance, we are not being consistent, as we pass this by value to the printClosureReport function... and in any case, the struct consists of an slice, that is a reference type in Go (consists of a pointer to something), so its already equivalent of passing as value...

Reply:
👍